osm2gtfs.py

```diff
@@ -31,18 +27,32 @@
 
 
 def main():
+
+    # Load config json file
+    try:
+        config = json.load(args.config)
```

Comment:
Making the config file argument required is not the fix I meant. I was suggesting to have `config.json` as a default argument for the config parameter, but don't include this file in the repository and even add it to `.gitignore`.

This enables people to have their local config file and play with it without it being tracked by version control and without having to specify the file every time they want to run the script. If people are happy with the official file in the repository, they can symlink `config-fenix.json` to `config.json`.